forms/setupdialog.ui

```diff
@@ -224,6 +239,9 @@ min-width: 200px;
    </item>
    <item>
     <widget class="QStackedWidget" name="wizardStackedWidget">
+     <property name="toolTip">
+      <string>Jump to machine key page</string>
+     </property>
```

Comment:
Sorry, I must have clicked on the wrong GUI element when adding this text.  I'll fix this and double-check it before submitting the new PR.